src/selection.ts

```diff
@@ -163,6 +181,12 @@ export interface IntervalSelectionConfig extends BaseSelectionDef {
    * appearance of the mark.
    */
   mark?: BrushConfig;
+
+  /**
+   * Initialize the selection with a mapping between field names and arrays of
+   * initial values.
+   */
+  init?: {[key: string]: SelectionInitArray};
```

Comment:
We have 

```
export interface SelectionInitMapping {
  [key: string]: SelectionInit;
}
```

Should this be define as `SelectionInitArrayMapping` too, for consistency?

Reply:
> field names?

(Or channels?)